configure.ac

```diff
@@ -354,6 +362,93 @@ if test $ac_cv_libevent_dir != "(system)"; then
   fi
 fi
 
+trylibssldir=""
```

Comment:
Remember to add a minimum version test in here

Reply:
I added the new TLS_server_method to the snippet below which fails when tried to build with any version prior to 1.1.0. I modified the error message accordingly. To test this, I ran a local docker with build (travis) commands. How should we go about the build?

Option 1. Update the base OS to bionic, install the standard packages which installs openssl 1.1.0g (March 2018)

Option 2. Keep the OS version as xenial (I saw the other request on this), download the openssl tarball (openssl-1.1.0j.tar.gz, Nov-2018), make and install it. With this option we have the freedom to control the version and go to even 1.1.1 if desired.

I verified both approaches locally with different docker images (bionic and xenial).

Reply:
I'm fine with either; I actually don't really care about the travis thing. I use a large set of buildbots that actually covers different platforms for release testing. The travis integration is just to help spot check PR's because people wanted it and I can't easily integrate buildbot for that. So travis isn't used to qualify releases, at any rate.